deployment-templates/compute-engine/forseti-instance.py

```diff
@@ -144,8 +146,7 @@ def GenerateConfig(context):
 
 # Install Forseti Security
 cd $USER_HOME
-rm -rf forseti-*
-rm -rf run_forseti.sh
+rm -rf *forseti*
```

Comment:
Yes, it's necessary. We should do a clean removal of all Forseti related files/directories.